src/modal/modal.scss

```diff
@@ -53,6 +53,7 @@
         white-space: nowrap;
         overflow: hidden;
         text-overflow: ellipsis;
+        line-height: normal;
```

Comment:
This keyword is reliant on the font family but usually means 1.2. Are you sure you want to use this?

What happened to the `align-items` fix you had?

Reply:
`align-items: stretch` resulted in the text not center aligning with the close button.

![Screen Shot 2022-02-28 at 1 43 03 PM](https://user-images.githubusercontent.com/849817/156039907-237d1da9-35cf-48aa-a4ed-a506c0db4744.png)

Reply:
What if add another layer of flex for centering `.iui-title`?

```suggestion
        align-self: stretch;
        display: flex;
        align-items: center;
```

Reply:
That does visually align but it breaks truncation.
![Screen Shot 2022-02-28 at 2 46 18 PM](https://user-images.githubusercontent.com/849817/156048577-b3752ead-e25a-4e58-8d89-7fe216401201.png)

[Looking into this right now](https://css-tricks.com/flexbox-truncated-text/) but not working on my initial try.

Reply:
Then maybe just use `line-height: 1.5`.

This is with `line-height: normal`:
![image](https://user-images.githubusercontent.com/9084735/156051125-ccf20ddb-689e-4099-add3-0f4235137c34.png)

This is with `line-height: 1.5` (notice the text has shifted down a bit, and is better centered)
![image](https://user-images.githubusercontent.com/9084735/156051159-dadd7407-c433-4730-8603-859515007803.png)

Reply:
Changed to `line-height: 1.5;`.